pandas/tests/test_strings.py

```diff
@@ -1603,6 +1603,18 @@ def test_pad_fillchar(self):
                                    "fillchar must be a character, not int"):
             result = values.str.pad(5, fillchar=5)
 
+    def test_pad_width(self):
```

Comment:
@jreback I'm relatively new to OOP in python, so could you let me know if there's anything I could do to improve my current implementation?
Thanks